.scripts/mocha_runner.js

```diff
@@ -4,15 +4,18 @@
 // Do not modify this file.
 // Use `.scripts/user/pretest.js instead`.
 
-require('babel-core/register');
-require('babel-polyfill');
+require('@babel/register');
+require('@babel/polyfill');
 
 // Add jsdom support, which is required for enzyme.
-var jsdom = require('jsdom').jsdom;
+var jsdom = require('jsdom');
```

Comment:
Could just do `const { JSDOM } = require('jsdom');`

But you should kill the `var` regardless